notification/notification_test.go

```diff
@@ -168,7 +167,11 @@ var _ = ginkgo.Describe("Notifications", ginkgo.Ordered, func() {
 		})
 
 		ginkgo.It("should have sent a notification for a config update", func() {
-			err := DefaultContext.DB().Model(&models.ConfigItem{}).Where("id = ?", config.ID).UpdateColumn("config", `{"color": "blue"}`).Error
+			event := models.Event{
+				Name:       "config.updated",
+				Properties: types.JSONStringMap{"id": config.ID.String()},
+			}
+			err := DefaultContext.DB().Create(&event).Error
```

Comment:
config.updated is only fired when we get a diff type in config_change table, so creating this manually